src/com/lftechnology/java/training/srijan/basics/genericclass/Util.java

```diff
@@ -0,0 +1,19 @@
+package com.lftechnology.java.training.srijan.basics.genericclass;
+
+
+
+public class Util {
+	
+	public static <K, V> boolean compare(Pair<K, V> p1, Pair<K, V> p2) {
+        return p1.getKey().equals(p2.getKey()) &&
+               p1.getValue().equals(p2.getValue());
+    }
+	
+	public static void main(String[] args) {
+		Pair<Integer, String> p1 = new Pair<>(1, "apple");
+		Pair<Integer, String> p2 = new Pair<>(2, "pear");
+		//Pair<Integer, String> p2 = new Pair<>(1, "apple");
+		boolean same = Util.<Integer, String>compare(p1, p2);
+		System.out.println(same);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Replace this usage of System.out or System.err by a logger. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS106)